recipes/xorg/all/conanfile.py

```diff
@@ -95,3 +95,9 @@ def package_info(self):
                      "xcb-xinerama", "xcb", "xkeyboard-config", "xcb-atom", "xcb-aux", "xcb-event", "xcb-util",
                      "xcb-dri3"]:
```

Comment:
```suggestion
                     "xcb-dri3", "uuid"]:
```

Reply:
beat me to it! thanks for the suggestion! :)